service/src/main/java/org/apache/griffin/core/job/CountDownTriggerListener.java

```diff
@@ -0,0 +1,49 @@
+package org.apache.griffin.core.job;
+
+import org.quartz.JobExecutionContext;
+import org.quartz.Trigger;
+import org.quartz.TriggerListener;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.concurrent.CountDownLatch;
+
+public class CountDownTriggerListener implements TriggerListener {
+    private static final Logger LOGGER = LoggerFactory
+            .getLogger(JobServiceImpl.class);
+    private CountDownLatch latch;
+    private String name;
+
+
+    public CountDownTriggerListener(CountDownLatch latch, String name) {
+        this.latch = latch;
+        this.name = name;
+    }
+
+    @Override
+    public String getName() {
+        return name;
+    }
+
+    @Override
+    public void triggerFired(Trigger trigger, JobExecutionContext context) {
+        LOGGER.info("CountDownTriggerListener triggerFired");
```

Comment:
should we add name of trigger or trigger listener here? otherwise, these info log messages are rather useless in production

Reply:
I removed it